use_case_examples/cifar_brevitas_training/bnn_pynq_train.py

```diff
@@ -71,7 +71,7 @@ def parse_args(args):
     )
     add_bool_arg(parser, "detect_nan", default=False)
     # Compute resources
-    parser.add_argument("--num_workers", default=4, type=int, help="Number of workers")
+    parser.add_argument("--num_workers", default=0, type=int, help="Number of workers")
```

Comment:
🙏🏼

Reply:
Have you tried this on a linux machine? I remember this failed on mine but it's probably fixed now? I can try if you want.

Reply:
yes if you could try that would be great, it fails on Mac if it's not set to 0

Reply:
Looks like it's working for me